bve-client/src/platform.rs

```diff
@@ -40,13 +40,13 @@ pub mod grabber {
     }
 
     impl Grabber {
-        pub fn new(window: &Window, grab: bool) -> Grabber {
-            let mut g = Grabber { grabbed: grab };
+        pub fn new(window: &Window, grab: bool) -> Self {
```

Comment:
Oh I guess I missed this last time

Reply:
Clippy did too, there is a lint for it but it didn't fire.